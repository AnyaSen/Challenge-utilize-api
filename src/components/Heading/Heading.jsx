import React from 'react';
import Styles from './Heading.module.scss';

export default function Heading({ text }) {
	return (
		<div className={Styles.HeadingContainer} id='programs'>
			<h2>{text}</h2>
		</div>
	);
}
