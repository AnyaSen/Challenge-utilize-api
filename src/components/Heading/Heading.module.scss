@import '../../assets/scss/breakpoints';
@import '../../assets/scss/variables';

.HeadingContainer {
	h2 {
		text-align: center;
		margin: $space-7;
	}
}
