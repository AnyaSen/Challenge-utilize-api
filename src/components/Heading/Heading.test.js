import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Heading from './Heading';

describe('<Heading />', () => {
	afterEach(cleanup);

	test('Should have correct text.', () => {
		const text = 'Button text';

		const { getByText, container } = render(<Heading text={text} />);

		const heading = getByText(text);

		expect(heading).toBeTruthy();
		expect(container.firstChild).toMatchSnapshot();
	});
});
