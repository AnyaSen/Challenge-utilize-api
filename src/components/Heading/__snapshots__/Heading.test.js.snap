// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> Should have correct text. 1`] = `
<div
  class="HeadingContainer"
  id="programs"
>
  <h2>
    Button text
  </h2>
</div>
`;
